rplidar_measure:
  max_tolerance_angle: 3
  MAXCDF: 1
  MAXPDF: 1
  MINPDF: 0
  group_margin: 5
  PID:
    dist: 
      p: 1
      i: 0
      d: 0
      set_dist: 0.6
    angle:
      p: 1.3
      i: 0
      d: 0
  lidar_target: "map"
  lidar_source: "track_link"
  rotation_angle: 3.1
  limit_x_speed: 0.2
  limit_y_speed: 0.1
  limit_z_angle_speed: 0.5

rplidar_pass_frame:
  max_tolerance_angle: 3
  MAXCDF: 1
  MAXPDF: 1
  MINPDF: 0
  group_margin: 5
  PID:
    dist: 
      p: 0.4
      i: 0
      d: 0
  limit_x_speed: 0.1
  limit_y_speed: 0.2
